config/owasp/suppressions.xml

```diff
@@ -32,4 +32,22 @@
     <packageUrl regex="true">^pkg:maven/org\.apache\.sling/org\.apache\.sling\.javax\.activation@.*$</packageUrl>
     <cve>CVE-2016-6798</cve>
   </suppress>
+  <suppress until="2020-11-30">
+        <notes><![CDATA[
+         file name: sscs-pdf-email-common-1.4.14.jar
```

Comment:
This suppress sscs-pdf-email-common-1.4.14.jar is not requied

Reply:
@srochani thanks sunil, removed.